tests/regression/nlevels_4_4_4_4/nlevels_4_4_4_4.cfg

```diff
@@ -44,7 +44,7 @@ optim_penalty_energy= 0.0
 optim_penalty_variation= 0.0
 optim_regul_tik0=false
 datadir = ./data_out
-output0 = expectedEnergy, population
+output0 = expectedEnergy, population, expectedEnergyComposite, populationComposite
```

Comment:
Do you want the tests to check these output files as well? Then I think we also need to add the reference output to `nlevels_4_4_4_4/base`